main.go

```diff
@@ -27,17 +27,47 @@ func newPromptUISearcher(items []string) list.Searcher {
 }
 
 func main() {
+	// if a value is passed as and argument and the value is not "version"
+	// we will assume it is the desired profile value and try to set it.
+	// If not argument is passed we will prompt the user to select a profile.
+	var desiredProfile = ""
+
 	if len(os.Args) > 1 && os.Args[1] == "version" {
 		fmt.Println("awsd version", version)
 		os.Exit(0)
+	} else if len(os.Args) > 1 && os.Args[1] != "version" {
```

Comment:
since we are checking len(os.Args) > 1 in both conditions, we could extract this into a parent condition and do something like this.

```go
if len(os.Args) > 1 {
	if os.Args[1] == "version" {
		fmt.Println("awsd version", version)
		os.Exit(0)
	} else {
		desiredProfile = os.Args[1]
	}
}
```

Reply:
done